### ISSUE TITLE
Nerf Pickpocketing

### ISSUE THREAD

**Issue by kepesk1** [Rotwood-Vale/Ratwood-Keep#291]
Pickpocketing is a bit chaotic and when the server is high pop, most people lose everything they're wearing before they can even blink. I would like for all pickpocketing levels to provide a notification to the target player, but the font is increasingly smaller based on the level of the pickpocket. Exclusively only legendary gets no notification of a pass or fail.

As well, make it so that pickpocketing within 15-30 seconds of the next pick pocket results in null EXP gain. This should stop self-pickpocketing off of your back to grind to legendary. It's basically abusing an unintended mechanic ... should also be punishable if caught but yea.

TL;DR pickpocketing is so annoying for people, help em out and make it so that pickpockets arent a meta
